CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 ## Unreleased
 
+## [2.7.0] - 2017-02-08
+### Added
+- backend: add "SSH dial timeout" to all backends, with a default of 5 seconds, configurable with `SSH_DIAL_TIMEOUT` backend setting
+- backend/docker: make the command to run the build script configurable with `BACKEND_DOCKER_EXEC_CMD` env var, default to `bash /home/travis/build.sh`
+- backend/gce: make it configurable whether to give a booted instance a "public" IP with `BACKEND_GCE_PUBLIC_IP`, defaults to `true`
```

Comment:
Ah, okay. I still think this should go in here since it's now a "documented" setting?